### PR TITLE
85 make tribes great again

### PR DESCRIPTION
**Pull request by RoadJDK**

### CODE REVIEW

**Comment by RoadJDK** [comment]
pls like + subscribe

File: marketplace/src/app/shared/post/post.component.ts [hunk 1]
```diff
@@ -51,6 +53,17 @@ export class PostComponent implements OnInit {
         console.log(this.filePath);
       }));
     }
+    this.checkTribe()
+  }
+
+  checkTribe() {
```
Comment (Akjo03) [inline marketplace/src/app/shared/post/post.component.ts:59]: Switch statements would be better :)

File: marketplace/src/app/pages/post/create/create.component.html [hunk 2]
```diff
@@ -20,9 +20,9 @@
             <div class="tribes">
                 <select formControlName="tribe" class="select">
                       <option selected>Alle tribes</option> <!-- TODO: MULTISELECT FUNCTIONALITY AND NGFOR* ALL OPTIONS-->
```
Comment (Akjo03) [inline marketplace/src/app/pages/post/create/create.component.html:22]: same for here

**Review by Akjo03 — DISMISSED** [review]
- Add ngFor and multiselect to both tribe selections.
- Implement switch statement for easier addition of tribes

File: marketplace/src/app/pages/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -15,9 +15,9 @@
     <div class="field">
       <select class="select" id="selectedTribe" formControlName="selectedTribe">
             <option selected>Alle tribes</option> <!-- TODO: NGFOR* ALL OPTIONS-->
-            <option value="1">One</option>
-            <option value="2">Two</option>
-            <option value="3">Three</option>
+            <option value="1">General</option>
```
[the remark this answers is not in the document]
Comment (RoadJDK) [reply marketplace/src/app/pages/dashboard/dashboard.component.html:18]: no I cba its good as it is :)

File: marketplace/src/app/pages/post/create/create.component.html (hunk 2 above)
Comment (RoadJDK) [reply marketplace/src/app/pages/post/create/create.component.html:22]: 🥇 nö

File: marketplace/src/app/shared/post/post.component.ts (hunk 1 above)
Comment (Akjo03) [reply marketplace/src/app/shared/post/post.component.ts:59]: no it would allow for easier addition of new tribes

Comment (Akjo03) [reply marketplace/src/app/shared/post/post.component.ts:59]: and it would change to a switch statement anyway

**Comment by RoadJDK** [comment]
https://www.urbandictionary.com/define.php?term=Yogurt%20Male

File: marketplace/src/app/pages/post/create/create.component.html (hunk 2 above)
Comment (Akjo03) [reply marketplace/src/app/pages/post/create/create.component.html:22]: ...

File: marketplace/src/app/pages/dashboard/dashboard.component.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Akjo03) [reply marketplace/src/app/pages/dashboard/dashboard.component.html:18]: ...

**Review by Akjo03 — CHANGES_REQUESTED** [review]
Fixed the thing with the switch statement.

**Review by Akjo03 — CHANGES_REQUESTED** [review]
:)

File: marketplace/src/app/pages/dashboard/dashboard.component.html [hunk 4]
```diff
@@ -15,9 +15,9 @@
     <div class="field">
       <select class="select" id="selectedTribe" formControlName="selectedTribe">
             <option selected>Alle tribes</option> <!-- TODO: NGFOR* ALL OPTIONS-->
```
Comment (Akjo03) [inline marketplace/src/app/pages/dashboard/dashboard.component.html:17]: Add ngFor here :)

File: marketplace/src/app/pages/post/create/create.component.html (hunk 2 above)
Comment (Akjo03) [inline marketplace/src/app/pages/post/create/create.component.html:22]: Add ngFor here too

**Review by Akjo03 — CHANGES_REQUESTED** [review]
Add the f*cking ngFors

**Comment by Akjo03** [comment]
will do this tomorrow evening, documents are more important

**Comment by Akjo03** [comment]
@jstnceronio could youreview this?

**Review by Akjo03 — APPROVED** [review]
äll good now in my opinion. @jstnceronio check the merge too

**Review by jstnceronio — APPROVED** [review]
SHillig
image

**Pull request merged**